provider/client.go

```diff
@@ -22,6 +22,11 @@ func NewClient(addr string, client *http.Client) *Client {
 	}
 }
 
+func (client *Client) GetInfo() (*core.ProviderInfo, error) {
+	// TODO: Implement
+	return nil, nil
```

Comment:
Maybe we should have this return an error stating that it's not implemented so we don't forget. Doesn't really matter, though.